examples/basic/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0"
+
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = ">= 1.49.0, < 2.0.0"
```

Comment:
Can we lock the basic example into 1.63.0 (as its the lowest supported version in the root level module). If we add an advanced example later, we can have it use latest provider version

Reply:
locked to 1.64.0